app/workers/mhv/account_statistics_job.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+module MHV
+  class AccountStatisticsJob
+    include Sidekiq::Worker
+
+    sidekiq_options retry: false
+
+    def perform
+      stats = { accounts_created_count: accounts_created.count,
+                existing_accounts_upgraded_count: existing_accounts_upgraded.count,
+                created_failed_upgrade_count: created_failed_upgrade.count,
+                created_and_upgraded_count: created_and_upgraded.count }
+      logger.info(mhv_account_statistics: stats)
```

Comment:
These could be sent directly to StatsD as counters, and then no parsing of log files would be needed. I wrote a doc here: https://github.com/department-of-veterans-affairs/vets.gov-team/blob/master/Work%20Practices/Engineering/Sending%20Metrics%20Using%20StatsD.md

I'd probably argue that these would be best as a gauge since we wouldn't be incrementing a counter the way that you would traditionally (even though this is a "count"). It'd be like this for each metric:

`StatsD.gauge('accounts_created_count', accounts_created.count)`